nox/virtualenv.py

```diff
@@ -275,21 +276,25 @@ def bin(self):
             return os.path.join(self.location, "bin")
 
     def create(self):
-        """Create the virtualenv."""
+        """Create the virtualenv or venv."""
         if not self._clean_location():
             logger.debug(
-                "Re-using existing virtualenv at {}.".format(self.location_name)
+                "Re-using existing virtual environment at {}.".format(
+                    self.location_name
+                )
             )
             return False
 
-        cmd = [sys.executable, "-m", "virtualenv", self.location]
+        cmd = [sys.executable, "-m", self.venv_or_virtualenv, self.location]
 
-        if self.interpreter:
+        if self.interpreter and self.venv_or_virtualenv == "virtualenv":
```

Comment:
What happens if they request a *different* interpreter version than the one Nox is installed on?

Reply:
Ah I think I see a bug. 

In the case of venv we should call
```
[self._resolved_interpreter, 'venv', '-m', 'self.location']
```
rather than use `sys.executable` since you can't specify the python interpreter to use in venv (it just uses the Python you create the venv with).

In the case of virtualenv it will remain unchanged
```
[sys.executable, 'virtualenv', '-m', 'self.location', '-p', self._resolved_interpreter]
```